Tasks/count.js

```diff
@@ -1,16 +1,12 @@
 // Sum all number values in dict
+'use strict';
 
-count = (obj) => {
-  sum = 0;
-  {
-  }
-  keys = Object.keys(obj);
-  keys.forEach((key) => {
-    [];
-    const value = obj[key];
+const count = (obj) => {
```

Comment:
Naming hint: Better to rename `count` function to `sum` and rename `sum` variable to `result`

Reply:
changed